runtime.md

```diff
@@ -545,6 +571,24 @@ procedure P()
          <posts> Ensures </posts>
          <args> PArgs </args>
          <rets> PRets </rets>
+
+    context #return HOLE
+    rule <k> #return X, Xs:ExprList => X, Xs ~> K </k>
+         <currentImpl> _ => N </currentImpl>
+         <procName> CurrentProc </procName>
+         <implStack> ListItem((N, K:K, Olds:Map, Locals:Map, Labels:Map)) => .List ... </implStack>
+         <olds> _ => Olds </olds>
+         <locals> _ => Locals </locals>
+         <labels> _ => Labels </labels>
+      requires isKResult(X ++ExprList Xs)
+
+    rule <k> #return .ExprList => K </k>
+         <currentImpl> _ => N </currentImpl>
+         <procName> CurrentProc </procName>
+         <implStack> ListItem((N, K:K, Olds:Map, Locals:Map, Labels:Map)) => .List ... </implStack>
+         <olds> _ => Olds </olds>
+         <locals> _ => Locals </locals>
+         <labels> _ => Labels </labels>
```

Comment:
These rules are also almost identical (only the first and last lines are different), but I'm also not sure here how to avoid duplicating code. 

The reason both are need right now is because if the procedure doesn't return any values, the `#call` should evaluate to `.K` rather than to an expression list.

Reply:
could we condense it to one rule, then have the first line be something like 
```
#return Xs => #if isEmpty(Xs) #then .K #else Xs #fi ~> K
```